crates/bevy_app/src/app.rs

```diff
@@ -135,9 +134,14 @@ pub struct SubApp {
     /// The [`SubApp`]'s instance of [`App`]
     pub app: App,
 
+    /// The schedule to run by default.
```

Comment:
Are systems no longer added to this by default? I think that was a helpful note.

Reply:
I am aware that the default schedule when adding a system will be removed with version `0.11.0`. #8079

Reply:
Ah right, so it will. Good fix then!